website/docs/add-secure-apps/flows-stages/stages/authenticator_webauthn/index.mdx

```diff
@@ -1,7 +1,9 @@
 ---
-title: WebAuthn authenticator setup stage
+title: WebAuthn / Passkeys authenticator setup stage
```

Comment:
@tanberry I’ve updated the other references to prevent confusion, but I believe we should include the term “passkey” in the title of the authenticator documentation since that’s what will appear in a Google search

Do you think that we can leave it this way or do you have any suggestions?